.eslintrc.js

```diff
@@ -17,21 +16,20 @@ module.exports = {
       "flowtype"
     ],
     "rules": {
-        "indent": [
-            "error",
-            2
-        ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ]
+      "no-magic-numbers": [
+        "off"
+      ],
+      "quotes": [
+        "error",
+        "single"
+      ],
+      "comma-dangle": [
+        "error",
+        "always-multiline"
```

Comment:
What I mean is that for personal preferences, things like single vs. double doesn't functionally matter at all for copy and pasting into other contexts.

If I take a snippet of what looks like an object here and paste it into es-lame browser code, it breaks. And since it's for a personal preference, I'd prefer we avoid.

Also @kenwheeler -- let's get in line on react-game-kit and react-music. 😉